airflow/bin/cli.py

```diff
@@ -1665,9 +1665,17 @@ def sync_perm(args):
             dag.access_control)
 
 
-Arg = namedtuple(
-    'Arg', ['flags', 'help', 'action', 'default', 'nargs', 'type', 'choices', 'metavar'])
-Arg.__new__.__defaults__ = (None, None, None, None, None, None, None)
+class Arg(object):
```

Comment:
What was wrong with this as a named tuple? Does mypy have trouble with namedtouples?

Reply:
`mypy` doesn't understand `namedtuple` default arguments set via `__new__.__defaults`, so when we instantiate `Arg` below with missing arguments, `mypy` complains. I think `mypy` understands the `namedtuple` default syntax in python >=3.6, but not this syntax.